docs/development/setup_development.rst

```diff
@@ -130,6 +130,10 @@ following ports:
 * Source Interface: `localhost:8080 <http://localhost:8080>`__
 * *Journalist Interface*: `localhost:8081 <http://localhost:8081>`__
 
+If you are using Tor to connect to your local Docker instance, make sure to set 
```

Comment:
let's either:

* move the [relevant text](https://docs.securedrop.org/en/release-0.12.2/development/tips_and_tricks.html#using-tor-browser-with-the-development-environment) from the tip page with the heading "Using Tor Browser with the Development Environment" to this page, i.e. in a new section before the `Setting Up a Multi-Machine Environment` section.
* Or just link out to the section https://docs.securedrop.org/en/release-0.12.2/development/tips_and_tricks.html#using-tor-browser-with-the-development-environment with some text like e.g. "You should use Tor Browser to test web application changes, see here for instructions.".

Reply:
Agreed, having DRY docs is as nice as having DRY code. @redshiftzero I took the second option and applied it here for your review.